dev-support/ci/selective_ci_checks.sh

```diff
@@ -434,6 +434,21 @@ function check_needs_findbugs() {
     start_end::group_end
 }
 
+function check_needs_native() {
```

Comment:
@adoroszlai Correct me if I am wrong this function will trigger native runs only if the changes are in 
`local pattern_array=(
        "^hadoop-ozone/dev-support/checks/native.sh"
        "^hadoop-hdds/rocks-native"
    )` right?

Reply:
Updated the patch to run native tests in all modules.